docs/lib/posixlib.rst

```diff
@@ -78,7 +78,7 @@ C Header          Scala Native Module
 `sys/uio.h`_      scala.scalanative.posix.sys.uio_
 `sys/un.h`_       N/A
 `sys/utsname.h`_  N/A
-`sys/wait.h`_     N/A
+`sys/wait.h`_     scala.scalanative.posix.sys.wait_
```

Comment:
@rwhaling  Thank you for the doc edit and your continuing energy.
Nothing is _ever_ simple

By visual execution only, I think the new link gives information but
will fail if anybody clicks on it.     

I think you will find that you got one of two places that need change.
I made the same mistake two or so weeks ago.  If you check my in
process PR, you will see the _two_ places I needed to change.
Not to puff up my own work, but a concrete example that I 
built.

Reply:
Got it!  Thank you for your patient explanation, will do

Reply:
@rwhaling It looks like you have addressed everything except the documentation change that is missing the link to the source code. @LeeTibbert Anything else?

Reply:
I think the missing/broken link is the only thing needed.  The rest of the PR
looks fine to my eyes.

Reply:
I think the missing/broken link is the only thing needed.  The rest of the PR
looks fine to my eyes.

Reply:
Thanks for taking a look at this again y'all, it totally slipped off my docket - I'll get it finished this weekend.